ambry-store/src/main/java/com.github.ambry.store/IndexSegment.java

```diff
@@ -54,17 +55,19 @@
  * lookup is performed to find key.
  */
 class IndexSegment {
-  private final static int Key_Size_Invalid_Value = -1;
-  private final static int Value_Size_Invalid_Value = -1;
-
-  private final static int Version_Field_Length = 2;
-  private final static int Key_Size_Field_Length = 4;
-  private final static int Value_Size_Field_Length = 4;
-  private final static int Crc_Field_Length = 8;
-  private final static int Log_End_Offset_Field_Length = 8;
-  private final static int Index_Size_Excluding_Entries =
-      Version_Field_Length + Key_Size_Field_Length + Value_Size_Field_Length + Log_End_Offset_Field_Length
-          + Crc_Field_Length;
+  private final static int KEY_SIZE_INVALID_VALUE = -1;
+  private final static int VALUE_SIZE_INVALID_VALUE = -1;
+
+  private final static int VERSION_FIELD_LENGTH = 2;
+  private final static int KEY_SIZE_FIELD_LENGTH = 4;
+  private final static int VALUE_SIZE_FIELD_LENGTH = 4;
+  private final static int CRC_FIELD_LENGTH = 8;
+  private final static int LOG_END_OFFSET_FIELD_LENGTH = 8;
+  private final static int LAST_MODIFIED_TIME_FIELD_LENGTH = 8;
+  private final static int RESET_KEY_TYPE_FIELD_LENGTH = 2;
+  private final static int INDEX_SIZE_EXCLUDING_ENTRIES =
```

Comment:
I don't think it is misleading though. New Index Segments will always refer to just one version i.e. the latest version of the PersistentIndex. Anyways, I am okay in taking up your suggestion